lib/sisimai/bite/email/messagelabs.rb

```diff
@@ -0,0 +1,213 @@
+              # This is the mail delivery agent at messagelabs.com.
+              #
+              # I was unable to deliver your message to the following addresses:
+              #
+              # maria@dest.example.net
+              #
+              # Reason: 550 maria@dest.example.net... No such user
+              #
+              # The message subject was: Re: BOAS FESTAS!
+              # The message date was: Tue, 23 Dec 2014 20:39:24 +0000
+              # The message identifier was: DB/3F-17375-60D39495
+              # The message reference was: server-5.tower-143.messagelabs.com!1419367172!32=
+              # 691968!1
+              #
+              # Please do not reply to this email as it is sent from an unattended mailbox.
+              # Please visit www.messagelabs.com/support for more details
+              # about this error message and instructions to resolve this issue.
+              #
+              #
+              # --b0Nvs+XKfKLLRaP/Qo8jZhQPoiqeWi3KWPXMgw==
+              # Content-Type: message/delivery-status
+              #
+              # Reporting-MTA: dns; server-15.bemta-3.messagelabs.com
+              # Arrival-Date: Tue, 23 Dec 2014 20:39:34 +0000
+              #
+              # Action: failed
+              # Status: 5.0.0
+              # Last-Attempt-Date: Tue, 23 Dec 2014 20:39:35 +0000
+              # Remote-MTA: dns; mail.dest.example.net
+              # Diagnostic-Code: smtp; 550 maria@dest.example.net... No such user
+              # Final-Recipient: rfc822; maria@dest.example.net
+              v = dscontents[-1]
+
+              if cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*([^ ]+)\z/)
+                # Final-Recipient: rfc822; maria@dest.example.net
+                if v['recipient']
+                  # There are multiple recipient addresses in the message body.
+                  dscontents << Sisimai::Bite.DELIVERYSTATUS
+                  v = dscontents[-1]
+                end
+                v['recipient'] = cv[1]
+                recipients += 1
+
+              elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                # Action: failed
+                v['action'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                # Status: 5.0.0
+                v['status'] = cv[1]
+
+              else
+                if cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                  # Diagnostic-Code: smtp; 550 maria@dest.example.net... No such user
+                  v['spec'] = cv[1].upcase
+                  v['diagnosis'] = cv[2]
+
+                elsif p =~ /\A[Dd]iagnostic-[Cc]ode:[ ]*/ && cv = e.match(/\A[ \t]+(.+)\z/)
+                  # Continued line of the value of Diagnostic-Code header
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + cv[1]
+                  havepassed[-1] = 'Diagnostic-Code: ' + e
+                end
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->